5.0/en/0x51-V51-OAuth2.md

```diff
@@ -68,7 +70,7 @@ In OpenID Connect flows, the "nonce" parameter provides CSRF protection. Otherwi
 
 OAuth 2.0 public clients utilizing the Authorization Code Grant are susceptible to the authorization code interception attack. Proof Key for Code Exchange (PKCE, pronounced "pixy") is the technique used to mitigate against the threat of authorization code interception attack.
 
-Originally, PKCE is intended to be used solely focused on securing native apps, but then it became a deployed OAuth feature. It does not only protect against authorization code injection attacks, but also protects authorization codes created for public clients as PKCE ensures that the attacker cannot redeem a stolen authorization code at the token endpoint of the authorization server without knowledge of the code_verifier.
+Originally, PKCE is intended to be used solely focused on securing native apps, but then it became a deployed OAuth feature. It does not only protect against authorization code injection attacks, but also protects authorization codes created for public clients as PKCE ensures that the attacker cannot redeem a stolen authorization code at the token endpoint of the authorization server without knowledge of the code_verifier. 
```

Comment:
@csfreak92 the way the second sentence reads to me here sort of implies that an attacker can do something other than an authorization code injection attack, if they get ahold of the authorization code. 

Suggestion to rework this paragraph to something such as this:
PKCE, initially designed to secure native apps, has evolved into a widely adopted OAuth feature even for web applications. It safeguards authorization codes for both public and confidential clients by ensuring that even if a code is stolen, it can't be redeemed at the authorization server's token endpoint without the corresponding code_verifier.